### PR TITLE
Action queue display

### PR DESCRIPTION
**Pull request by samholmes**
#### Dependencies
https://github.com/EdgeApp/edge-react-gui/pull/3532
https://github.com/EdgeApp/edge-react-gui/pull/3526

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

### CODE REVIEW

**Comment by Jon-edge** [comment]
Need to run the precommit hook to generate the enUS.json

File: src/locales/strings.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import { getLocales } from 'react-native-localize'
+// import { getLocales } from 'react-native-localize'
```
Comment (Jon-edge) [inline src/locales/strings.js:3]: What is the reason for removing this and leaving it commented?

File: src/plugins/action-queue/cleaners.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    }
+
+    // Invalid type
+    throw new TypeError('Expected Error')
+  },
+  // Serialize as a JsonError
+  asJsonError
+)
+
+export const asActionOp: Cleaner<ActionOp> = asEither(
```
Comment (Jon-edge) [inline src/plugins/action-queue/cleaners.js:38]: Not in the scope of this review, but this is currently a Flow error. Does there need to be a change to package.json to support the extension of asEither?

Comment (Jon-edge) [reply src/plugins/action-queue/cleaners.js:38]: Disregard, just heard you have a cleaner PR out.

**Comment by Jon-edge** [comment]
Few minor comments, looks awesome!

File: src/locales/strings.js (hunk 1 above)
Comment (samholmes) [reply src/locales/strings.js:3]: This was only during testing, should of not gotten into the commit. Fixed.

Comment (Jon-edge) [reply src/locales/strings.js:3]: Didn't get updated?

**Review by Jon-edge — APPROVED** [review]

**Pull request merged**